### DISCUSSION TITLE
using another model for values passed to accepted_values

### DISCUSSION THREAD

**Discussion by epapineau** [dbt-labs/docs.getdbt.com#1479] [Archive]
Is there an `accepted_values`-type test where the accepted values can be set using the values of a column in another model?

**Comment by gwenwindflower** [comment]
Yes! It will shock and surprise you!

So if we step back and look at the operation: what we need to  do really is check that all the individual values in a column correspond to a set of unique values in a different column, often in a different table. We have a test that does exactly this!

[Relationships!](https://docs.getdbt.com/reference/resource-properties/tests#relationships)

We can use the `relationships` test to create a sort of 'dynamic' accepted values test. I've even gone so far as to recommend if you have more than 10 accepted values, to consider putting them in a seed and using relationships.
